engine/object.asm

```diff
@@ -108,6 +108,8 @@ ObjectLoop2@
             std         COB.statePtr,x          * the state data pointer is now correct
             leau        7,u                     * point to the initialization data for this object in the object init stream
             ldy         COB.odtPtr,x
+            lda         [ODT.vpageAddr,y]       * load the page number
```

Comment:
Can this be surrounded by an "IFNE OBJPAGES-1" macro?

Reply:
Yes!

Reply:
Fixed